tests.live/Run.ps1

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+param (
+    [Parameter()]
+    [switch]
+    $SkipInstall=$False
+)
+
+if (-not $SkipInstall) {
+    .\Install-Artifacts.ps1
+}
+
+# For debug, print all relevant environment variables:
+Get-ChildItem env:AZURE*
+Get-ChildItem env:*VERSION
```

Comment:
In some cases on CI, tables don't get formatted correctly, such that you may need to use `Format-Table` to force things to show up right in CI.
```suggestion
Get-ChildItem env:AZURE*, env:*VERSION | Format-Table | Out-String | Write-Host
```